### PR TITLE
Setproctitle 1.1.9

### PR DESCRIPTION
**Pull request by ocefpaf**
Adding `setproctitle`.

### CODE REVIEW

**Comment by scopatz** [comment]
Looks good to me!

**Comment by scopatz** [comment]
I don't understand where this error is coming from:

```
Building  setproctitle-1.1.9-py34_0 
Traceback (most recent call last): 
  File "C:\Miniconda\Scripts\obvci_conda_build_dir-script.py", line 9, in <module> 
    load_entry_point('Obvious-ci==0.6.1', 'console_scripts', 'obvci_conda_build_dir')()
  File "C:\Miniconda\lib\site-packages\obvci\cli\conda_build_dir.py", line 19, in main 
    return Builder.handle_args(args).main()
  File "C:\Miniconda\lib\site-packages\obvci\conda_tools\build_directory.py", line 239, in main
    self.build(meta) 
  File "C:\Miniconda\lib\site-packages\obvci\conda_tools\build_directory.py", line 217, in build 
    build.build(meta.meta) 
  File "C:\Miniconda\lib\site-packages\obvci\conda_tools\build.py", line 23, in build
    build_module.build(meta, verbose=False, post=None) 
TypeError: build() got an unexpected keyword argument 'verbose'
```

**Comment by ocefpaf** [comment]
Latest conda-build broke `obvious-ci`  :unamused:

**Comment by ocefpaf** [comment]
I added https://github.com/conda-forge/staged-recipes/commit/992c5457c3f257900561c247d731f139394df2c4 just to test things here.  We don't want to  pin conda-build version number.

What we can do:

- Fix `obvious-ci`
- Stop adding recipes here and create feedstocks
- Start using conda-build-all

**Comment by scopatz** [comment]
I think the best option is the Fix `obvious-ci`.  Unless things have changed recentl, creating feedstocks still requires a lot of manual effort that isn't worth it for small packages.  

How does build-all work in this case?

**Comment by ocefpaf** [comment]
> How does build-all work in this case?

It should work as "drop-in replacement for `obvious-ci`, right @pelson?  I will take a look at this tomorrow.  Time to :sleeping:

File: recipes/setproctitle/bld.bat [hunk 1]
```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install
```
Comment (pelson) [inline recipes/setproctitle/bld.bat]: We need to fix this. https://github.com/conda/conda-build/issues/682

I'll try to get around to it in the near future!

Comment (ocefpaf) [inline recipes/setproctitle/bld.bat]: Yeah...  It is really ugly (and inconsistent) to remove `build.sh` and leave `bld.bat`.

**Comment by scopatz** [comment]
Woohoo Thanks for merging this.

**Pull request merged**